packages/terra-doc-template/src/DocTemplate.jsx

```diff
@@ -86,6 +86,12 @@ const DocTemplate = ({ packageName, readme, srcPath, examples, propsTables, ...c
     customProps.className,
   ]);
 
+  let exampleHeader;
+  if (localExamples.length > 0) {
+    const exampleHeaderText = localExamples.length > 1 ? 'Examples' : 'Example';
+    exampleHeader = <h1 className={cx('.examples-header')}>{exampleHeaderText}</h1>;
+  }
+  
```

Comment:
This is whitespace on this line that is causing a lint error. Depending on which code editor you are using, you can set it up to trim whitespace on file save.